app/controllers/result_filters/provider_controller.rb

```diff
@@ -0,0 +1,29 @@
+module ResultFilters
+  class ProviderController < ApplicationController
+    include FilterParameters
+
+    def new
+      if params[:query].blank?
+        flash[:error] = "Training provider"
+        return redirect_back
+      end
+
+      @provider_suggestions = Provider
```

Comment:
Worth moving to a scope?

Reply:
I don't believe JSONApiClient supports scopes, and as it stands this is only used here